### ISSUE TITLE
Using SWRevealViewController with multiple storyboards

### ISSUE THREAD

**Issue by abbeyjackson** [John-Lluch/SWRevealViewController#464]
Hi, how can one move to a view controller on another storyboard and maintain the side reveal? 

The methods I know for jumping between storyboards leave me with no side reveal button.

```
UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"StoryboardName" bundle:nil]; 
UIViewController *viewController = [storyboard instantiateViewControllerWithIdentifier:@"ViewControllerName"];
[self presentViewController:viewController animated:YES completion:nil];
```

What do I need to do differently to keep the side reveal?

**Comment by tirrorex** [comment]
Hi, here is my code in swift :  
func initStoryboard(controller: UIViewController, storyboardName: String)
    {
        let storyboard = UIStoryboard(name: storyboardName, bundle: nil)
        let childController = storyboard.instantiateInitialViewController() as UIViewController!
        addChildViewController(childController)
        childController.view.backgroundColor = UIColor.redColor()
        controller.view.addSubview(childController.view)
        controller.didMoveToParentViewController(childController)
    }

so basically you don't present the view controller, you get the view from the other storyboard and you add it as a subview, this will allow you to keep the side reveal

**Comment by abbeyjackson** [comment]
Thanks this is a big help! :+1:

**Comment by kishorehvk** [comment]
Hi can u share me the objective c code for the above swift code ... Thanks In advance

**Comment by chrisjmccrum** [comment]
@abbeyjackson did this work for you? I'm trying to figure out how to implement this fix if you don't mind helping me out
